com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -579,6 +599,50 @@ private static Project createProject(String rootDirectory, String buildDirectory
         return project;
     }
 
+    /**
+     * Set the log level to use for the root logger and all handlers
+     * @param level The java.util.logging.Level to use
+     */
+    public static void setLogLevel(Level level) {
+        Logger rootLogger = LogManager.getLogManager().getLogger("");
+        rootLogger.setLevel(level);
+        for (Handler h : rootLogger.getHandlers()) {
+            h.setLevel(level);
+        }
+    }
+
+    /**
+     * Enable or disable verbose logging
+     * @param enable Set to true to enable verbose logging
+     */
+    public static void setVerboseLogging(boolean enabled) {
+        Bob.setLogLevel(enabled ? Level.CONFIG : Level.WARNING);
+    }
+
+    /**
+     * Remove the log stream handler from the root logger
+     * Use this together with addLogStream()
+     */
+    public static void removeLogStream() {
+        if (logStreamHandler != null) {
+            LogManager.getLogManager().getLogger("").removeHandler(logStreamHandler);
+            logStreamHandler = null;
+        }
+    }
+
+    /**
+     * Add a log stream handler to the root logger. Use this to send log records
+     * from bob to some other source, for instance the Defold editor. If a log
+     * stream has already been added it will first be removed before the new one
+     * will be added
+     * @param out The output stream to write log records to
+     */
+    public static void addLogStream(OutputStream out) {
```

Comment:
I think this should be called `setLogStream` because `add` implies there might be multiple log streams, while the implementation only allows one by removing the previous log stream handler.